packages/busybox/busybox.config

```diff
@@ -1002,22 +1002,22 @@ CONFIG_FEATURE_WGET_TIMEOUT=y
 # CONFIG_FEATURE_WGET_OPENSSL is not set
 CONFIG_WHOIS=y
 # CONFIG_ZCIP is not set
-# CONFIG_UDHCPD is not set
+CONFIG_UDHCPD=y
```

Comment:
I'm not sure `udhcpd` would work without root, or at all on Android.

Reply:
As for udhcpd, if one condition is met, it can be used with non-root permissions, but not on non-rooted devices.
This condition is that the kernel parameter net.ipv4.ip_unprivileged_port_start is equal to 1, which can be automatically modified by the service.sh function of Magisk/KSU/APatch.